packages/renderer/src/lib/dashboard/NotificationsBox.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+import { onMount, onDestroy } from 'svelte';
+import type { Unsubscriber } from 'svelte/motion';
+import type { Notification } from '../../../../main/src/plugin/api/notification';
+import NotificationCard from './NotificationCard.svelte';
+import { notificationQueue } from '/@/stores/notifications';
+
+let notificationsUnsubscribe: Unsubscriber;
+let notifications: Notification[] = [];
+
+onMount(async () => {
+  notificationsUnsubscribe = notificationQueue.subscribe(value => {
+    // take only first 3 elements of the notifications queue
+    notifications = value.slice(0, 3);
+  });
+});
+
+onDestroy(() => {
+  if (notificationsUnsubscribe) {
+    notificationsUnsubscribe();
+  }
+});
+</script>
+
+<div class="bg-charcoal-800 m-5 px-5 py-4 rounded-lg">
+  <div class="flex flex-col items-center justify-content space-y-3">
+    <span class="text-lg text-white mb-1"> New Notifications </span>
+    {#if notifications.length === 0}
+      <div class="pt-3 pb-5" aria-label="No notification to display">No notification to display</div>
```

Comment:
I would suggest the following classes on the no notification to display message:

text-gray-700
text-sm